infrastructure/ansible/roles/lobby_database/tasks/main.yml

```diff
@@ -20,6 +20,7 @@
       - postgresql
       - postgresql-contrib
       - libpq-dev
+      - python-psycopg2
```

Comment:
Is this python 2 dependency really needed? :(
python 2 is supposed to be end of life at the end of this year

Reply:
Yes, I was tempted to remove the python 3 dependency, though that limits the number of environments we can deploy to. Vagrant environment uses the python 3 version. Ubuntu 18.04 LTS needed the python 2 dep.